src/sdk/uservarmanager.cpp

```diff
@@ -63,88 +63,31 @@ const wxChar *bim[] =
     _T("cflags"),
     _T("lflags")
 };
-const wxArrayString builtinMembers((size_t) 7, bim);
+const wxArrayString UserVariableManager::builtinMembers((size_t) 7, bim);
 
-class GetUserVariableDialog : public wxScrollingDialog
+class UserVarManagerDefaultUI : public UserVarManagerUI
```

Comment:
The name of this doesn't designate what it actually does...

Reply:
Name changed in e9f17b2